app/src/main/java/com/example/seoulgonggong/ui/filter/FilterCustomView.kt

```diff
@@ -0,0 +1,83 @@
+package com.example.seoulgonggong.ui.filter
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.example.seoulgonggong.R
+import com.example.seoulgonggong.databinding.CustomFilterBinding
+import com.google.android.material.chip.Chip
+import com.google.android.material.chip.ChipGroup
+import com.google.android.material.shape.CornerFamily.ROUNDED
+import com.google.android.material.shape.ShapeAppearanceModel
+
+class FilterCustomView(context: Context, attr: AttributeSet? = null) :
+    ConstraintLayout(context, attr) {
+    private val binding: CustomFilterBinding
+    private val chips: MutableList<Chip> = mutableListOf()
+    private var chipGroup: ChipGroup
+
+    init {
+        val inflater = LayoutInflater.from(context)
+        binding = CustomFilterBinding.inflate(inflater, this, true)
+        chipGroup = binding.cgFilterGroup
+    }
+
+    fun setFilterTitle(title: String) {
+        binding.tvFilterTitle.text = title
+    }
+
+    fun addFilterOptionGroup(group: List<String>) {
+        group.forEach {
+            addFilterOption(it)
+        }
+    }
+
+    private fun addFilterOption(option: String) {
+        val chip = Chip(context)
+        setChipSize(chip)
+        chip.text = option
+        chip.isCheckable = true
+        setChipStyle(chip)
+        chipGroup.addView(chip)
+        chips.add(chip)
+    }
+
+    private fun setChipSize(chip: Chip) {
+        chip.setPadding(16, 8, 16, 8)
+        val layoutParams =
+            ViewGroup.LayoutParams(
+                ViewGroup.LayoutParams.WRAP_CONTENT,
+                ViewGroup.LayoutParams.WRAP_CONTENT,
+            )
+        chip.layoutParams = layoutParams
+    }
+
+    private fun setChipStyle(chip: Chip) {
+        chip.setChipBackgroundColorResource(R.color.chip_bg_color)
+        chip.setTextColor(resources.getColorStateList(R.color.chip_text_color, null))
+        chip.setChipStrokeColorResource(R.color.main_teal)
+        chip.chipStrokeWidth = 5F
+        chip.shapeAppearanceModel =
+            ShapeAppearanceModel.builder()
+                .setAllCorners(ROUNDED, 50F)
+                .build()
+        chip.setTextAppearance(R.style.subtitle_2)
+        chip.chipIcon = null
+    }
+
+    fun getSelectedOptions(): List<String> {
+        val selectedOptions = mutableListOf<String>()
+        chips.forEach {
+            if (it.isChecked) {
+                selectedOptions.add(it.text.toString())
+            }
+        }
+        return selectedOptions
```

Comment:
```suggestion
        return chips.filter { it.isChecked }.map { it.text.toString() }
```
이렇게 해줘도 될 것 같아요!
(오늘 내 pr에 써준 뽀또 리뷰 훔쳐오기ㅎㅎㅎ)

Reply:
꺄올 활용력 짱😘 저도 후다닥 고쳤슴당!ㅋㅋㅋ